presto-orc/src/main/java/com/facebook/presto/orc/Filters.java

```diff
@@ -34,7 +35,7 @@
 
     private Filters() {}
 
-    private static class AlwaysFalse
+    public static class AlwaysFalse
```

Comment:
this class should stay private

Reply:
These classes need to stay private. Use alwaysFalse().equals(filter) instead of filter instanceof AlwaysFalse.